City-of-Helsinki/drupal-asuntotuotanto
# SATT-38: Hitas and Haso listing page cache changes

### Description

Hitas and Haso cache fix which also fixes hitas and haso listing filters

### How to test it

- make fresh
- login as admin
- go re-indexing apartment listing index https://asuntotuotanto.docker.so/fi/admin/config/search/search-api/index/apartment_listing
- now open a new browser or incognito 
- go haso or hitas page https://asuntotuotanto.docker.so/fi/asuntohaku/hitas
  - check that listing page is working
  - check that filters are showing some results
- now go back in that browser where you are logged in
- first edit some project like adding some name changes
  - then edit one of that project apartment maybe just clicking save
  - and remember which project and type (hitas or haso)
- now go search api apartment listing index https://asuntotuotanto.docker.so/fi/admin/config/search/search-api/index/apartment_listing
  - check that the indexing is out of date so x/x indexed is not matching and you can press index now
  - if x/x is matching then you need to edit again some apartment and change something value
 - press index now
 - now on that other browser or incognito mode
   - Go hitas / haso page and check that your changes are showing on listing pages
